### PR TITLE
Implemented Story 4 and 5

### PR DESCRIPTION
**Pull request by kvptkr**
Story 4 AC: 
- An API route for the homepage is implemented
- The rendered view contains a sign up button and log in button for both cleaners and employing companies
- A synopsis of what AOC is provided in text

Story 5 AC:
- Clicking sign-up button on landing page renders sign-up page
- Signup page contains submit button (saves sign up info in future sprint, but out of scope for this sprint)
- Upon clicking submit button, “Create Profile” page is rendered
- The profile creation page is not currently gated by login authentication (In future sprints, only users with AOC account will be able to create profile)

### CODE REVIEW

**Comment by christinalee98** [comment]
1. Make sure you run elements through the auto-formatter
2. How did you create the button component on the sign-up page? I cannot seem to find it

**Review by christinajiyunlee — APPROVED** [review]
1. Make sure you run elements through the auto-formatter
2. How did you create the button component on the sign-up page?

File: UI/aoc-app/src/App.js [hunk 1]
```diff
@@ -1,13 +1,34 @@
...
+    <Router>
+      <div className="App">
+        <Switch>
+          <Route exact path="/">
+              <HomePage/>
+            </Route>
+            <Route exact path="/createProfile">
+              <CreateProfile />
+            </Route>
+            <Route exact path="/sign-up">
```
Comment (matthewfollegot) [inline UI/aoc-app/src/App.js]: Let's make the routes standardized to either kebab-case or camel-case. Either is fine, but they should follow the same convention

File: UI/aoc-app/src/Components/HomePage.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            <div className = "buttons">
+                <Link to="/createProfile">  <Button color="primary onClick{() => }">Sign-Up</Button> </Link>
+                <Button color="secondary">Login</Button>
+            </div>
+        </React.Fragment>
+    );
+}
+
+
+export default HomePage;
```
Comment (matthewfollegot) [inline UI/aoc-app/src/Components/HomePage.jsx]: nit: newline at EOF

File: UI/aoc-app/src/Components/SignUpForm.jsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Button, Col, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
```
Comment (matthewfollegot) [inline UI/aoc-app/src/Components/SignUpForm.jsx]: Remove FormText if it isn't being used. Similar to what you commented on my PR :)

File: UI/aoc-app/src/Components/CreateProfile.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Button, Col, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
```
Comment (matthewfollegot) [inline UI/aoc-app/src/Components/CreateProfile.jsx]: Unused FormText

File: UI/aoc-app/src/Components/SignUpForm.jsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+                <Button>Submit</Button>
+                </Col>
+            </FormGroup>
+        </Form>
+       
+    );
+}
+
+
+export default SignUpForm;
```
Comment (matthewfollegot) [inline UI/aoc-app/src/Components/SignUpForm.jsx]: nit: newline

File: UI/aoc-app/src/Components/SignUpForm.jsx [hunk 6]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
```
Comment (matthewfollegot) [inline UI/aoc-app/src/Components/SignUpForm.jsx]: this file seems to just be a duplicate of `CreateProfile.jsx`. This should render that view when a cleaner creates an account through OAuth

File: UI/aoc-app/src/App.js (hunk 1 above)
Comment (kvptkr) [reply UI/aoc-app/src/App.js]: Yeah, that's a good point. According to this stack link, hyphenated is the way the go: 
https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris

File: UI/aoc-app/src/Components/SignUpForm.jsx (hunk 3 above)
Comment (kvptkr) [reply UI/aoc-app/src/Components/SignUpForm.jsx]: This is actually from your commit! I had to cherry-pick your commit on top and see that this

File: UI/aoc-app/src/Components/CreateProfile.jsx (hunk 4 above)
Comment (kvptkr) [reply UI/aoc-app/src/Components/CreateProfile.jsx]: see above!

File: UI/aoc-app/src/Components/SignUpForm.jsx (hunk 6 above)
Comment (kvptkr) [reply UI/aoc-app/src/Components/SignUpForm.jsx]: Yeah, I acidentally staged this file. I'll remove and go from there.

File: UI/aoc-app/src/Components/SignUpForm.jsx (hunk 3 above)
Comment (matthewfollegot) [reply UI/aoc-app/src/Components/SignUpForm.jsx]: Ya I know, I took your feedback before I merged, so you'll have a merge conflict if you don't remove it.

File: UI/aoc-app/src/App.js (hunk 1 above)
Comment (kvptkr) [reply UI/aoc-app/src/App.js]: I think that we should add a story to our backlog for a little refactor for all of this route stuff, so I'm gonna hold off for now. I have a few ideas, let's discuss at standup.

File: UI/aoc-app/src/Components/SignUpForm.jsx (hunk 3 above)
Comment (kvptkr) [reply UI/aoc-app/src/Components/SignUpForm.jsx]: I'm actually planning on dropping that commit and then rebasing on main, so I don't think that it should be an issue.

**Pull request merged**